blog/_posts/2016-10-19-scalajs-bundler.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: "scalajs-bundler has been released"
```

Comment:
"Use npm packages in Scala.js with scalajs-bundler"? The title is quite important for search engines so pick those words carefully!